pyproject.toml

```diff
@@ -67,13 +106,13 @@ ignore = [
 
 
     # Stricter type checking rules that that are disabled.
-    "A003",  # Class attribute shadowing a python bulitin (class attributes are seldom if ever referred to bare)
```

Comment:
codespell

Reply:
heh this one's funny; can you update it to `built-in` please?